### PR TITLE
Create Reference Data Search Product Description.md

### PR DESCRIPTION
**Pull request by mshukla1**

### CODE REVIEW

File: Proposed product descriptions/Reference Data Search Product Description.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    sequenced for the same species and strain, metadata is absolutely
+    crucial to distinguish one genome from another and understand why
+    the genome was sequenced. Often, filtering the genome based on
+    available metadata is essential first step to identify a set of
+    genomes one wants to include in comparative genomic analysis. For
+    example, a user may want to select Brucella genomes isolated from
+    different hosts and compare them using pan-genome tool to understand
+    host-specific adaptations.
+
+-   **Value added annotations:** One of the objectives of the KBase
```
Comment (dangunter) [inline Proposed product descriptions/Reference Data Search Product Description.md:83]: This is a dependency on a possibly deferred item. Need to account for this.

File: Proposed product descriptions/Reference Data Search Product Description.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    operons, which provide rich source of information and form the basis
+    of many comparative genomic analysis methods. As we ad support for
+    value added annotations in the KBase genome annotation method, we
+    will also incorporate them into the object models, Solr schema, and
+    the reference data search.
+
+### Future work
+
+1.  Enhance data landing pages for genomes and genomic features.
+
```
Comment (dangunter) [inline Proposed product descriptions/Reference Data Search Product Description.md:96]: Enhance them how? This seems out of scope.

File: Proposed product descriptions/Reference Data Search Product Description.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+-   Display additional feature annotations
+
+### 
+
+### Timeline for feature release
+
+**Sprint 1:**
+
+-   Select genomes and genomic features from search results and save
+    them as genome or feature sets.
```
Comment (dangunter) [inline Proposed product descriptions/Reference Data Search Product Description.md:108]: Should describe planned path to use of new genome types

File: Proposed product descriptions/Reference Data Search Product Description.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+**Sprint 2:**
+
+-   Add biochemistry data to reference data search
+
+-   Design and develop data landing pages for the biochemistry data
```
Comment (dangunter) [inline Proposed product descriptions/Reference Data Search Product Description.md:114]: Seems out of scope for search.

**Comment by aparkin** [comment]
@dangunter why is it out of scope for search? Is a landing page one of the final results of search?

File: Proposed product descriptions/Reference Data Search Product Description.md (hunk 2 above)
Comment (aparkin) [inline Proposed product descriptions/Reference Data Search Product Description.md:96]: We should discuss how much of the value added analysis is in scope for Q2 in any case. Something to consider given we want these analyses ultimately to be in SDK format and easily extensible and there are issues with out to schedule and scale. Worth discussion. 

@dangunter it seems we have to discuss how to factor getting reference data into KBase, making it searchable and displaying the results... thus your comments about Landing pages. I agree anything done on landing pages should be specified. But is there a specific concern other than it might be biting off too much?

**Comment by dangunter** [comment]
So of course I don't disagree that landing pages and search are related. Connecting search in general to landing pages in general should be in scope, and demonstrating with some existing landing pages, called out in the PD, certainly makes sense.

But creating entirely new type-specific landing pages, especially for your own group's new types, seems like it should be fully described and scoped as a separate sprint that is (a) focused on those types and (b) should consider, at that time, the linkage(s) to the Data API. I do not oppose the landing pages, I am just saying it should not be in a PD that your email summarized as "Get the reference data update process running again in an automated fashion."

If the powers that be decide these need to happen in parallel for more personnel-related reasons, I would ask at least that the title and summary be modified to clarify that this PD is also for developing landing pages for the biochemistry types. Again, in that case, at least describing how this would be done in a Data API should be in scope (and the data team, i.e. @mlhenderson, should be pulled into the loop).

**Comment by aparkin** [comment]
There seems to be two topics here I think. 

1) If a PD becomes too big it is hard to see how it will factor into sprints, etc. There are a lot of moving parts both here and in other PDs. It might be good to make a product a series of "point" deliverables- so in this case a PD for each functional object (genomes, biochemistry, etc). Maybe even acknowledging the need to deliverable a standard framework for automated update; a framework for application of value added analyses; and communication of update to users, etc. So maybe this needs to be broken into a few PDs?

2) Remember that a PD is a description of a desired effect rather than an implementation plan. @dangunter -- You and Matt  are part of the implementation team and thence should be part of the implementation plan for a possible PD. Ideally, Shane would be coordinating an implementation team review of the PD and talking about how to break them into sprints and who implements and how. So the idea that the product that designs biochemistry landing pages would trigger a discussion of implementation of Data API extensions, and application of these to create the page is certainly an important part of this process. Product owners should be cognizant of this. They are to specify what things should do and perhaps how they look to users. Implementation team is supposed to help design how it works and how to stage the work.

**Comment by scanon** [comment]
@mshukla1 It seems like some of this has already been done but other pieces haven't.  Do you mind cleaning up this PD so I can merge it?
